.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [windows-2022]
+        config: [Debug, Release]
+        platform: [x64, ARM64]
+        include:
+          - platform: x64
+            cmake_platform: x64
+          - platform: ARM64
+            cmake_platform: ARM64
+
+    runs-on: ${{ matrix.os }}
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v2
+
+    - name: Configure CMake
+      run: |
+        cmake -B build -A ${{ matrix.cmake_platform }} -DCMAKE_BUILD_TYPE=${{ matrix.config }} -DKF_BUILD_TESTS=OFF
+
+    - name: Build
+      run: |
+        cmake --build build --config ${{ matrix.config }} --parallel
+
+  build-with-tests:
```

Comment:
- remove the build-with-tests job
- in the build job remove `-DKF_BUILD_TESTS=OFF`
- windows-2022 image has WDK preinstalled, so no additional steps are required